src/test/series-utils.js

```diff
@@ -22,7 +22,6 @@ import test from 'tape';
 import 'babel-polyfill';
 
 import React from 'react';
-import equal from 'deep-equal';
 
 import {
```

Comment:
I think it's a good time to add a test for the `getStackedData` since it's quite complex looking.